examples/resources/cluster/resource.tf

```diff
@@ -1,27 +1,42 @@
-resource "risingwavecloud_cluster" "example" {
-  name    = "dev"
-  version = "v1.3.0"
-  region  = "us-east-2"
-  resourcev1 = {
+resource "risingwavecloud_cluster" "my_cluster" {
+  region  = "us-east-1"
+  name    = "my_cluster"
+  version = "v1.8.0"
+  spec = {
     compute = {
-      type    = "p-1c2g"
-      replica = 1
+      default_node_group = {
+        cpu     = "2"
+        memory  = "8 GB"
+        replica = 1
+      }
     }
     compactor = {
-      type    = "p-2c4g"
-      replica = 3
-    },
+      default_node_group = {
+        cpu     = "1"
+        memory  = "4 GB"
```

Comment:
What If the cpu/memory values are invalid?

Reply:
The error message will contain the available settings: https://github.com/risingwavelabs/terraform-provider-risingwavecloud/pull/20/files#diff-39b6750b80d05e2b19ce3ccfbc3c89b47161d14d65e76a22605a1cb434fb0d7cR287